x/incentive/keeper/btc_staking_gauge.go

```diff
@@ -113,3 +139,45 @@ func (k Keeper) btcStakingGaugeStore(ctx context.Context) prefix.Store {
 	storeAdapter := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
 	return prefix.NewStore(storeAdapter, types.BTCStakingGaugeKey)
 }
+
+func (k Keeper) SetFPDirectGauge(ctx context.Context, height uint64, gauge *types.Gauge) {
```

Comment:
Suggestion to use the collections map in keeper `collections.Map[[]byte, types.Gauge]` but no strong opinion, if you want to go with store just like the other gauge, all good to me as well

Reply:
yes, I was following same style as the other gauge